### PR TITLE
Fix ARM64 arch name for rpm.

### PR DESCRIPTION
**Pull request by MrMYHuang**
This PR breaks arm64 deb:
https://github.com/TheAssassin/AppImageLauncher/pull/509
This PR fixes it.

### CODE REVIEW

File: ci/build.sh [hunk 1]
```diff
@@ -107,10 +111,10 @@ make -j "$nproc"
 cmake .
 
 # build Debian package
-cpack -V -G DEB
+CPACK_RPM_PACKAGE_ARCHITECTURE="${ARCH}" cpack -V -G DEB
```
Comment (TheAssassin) [inline ci/build.sh:116]: Looks like a typo.

Comment (MrMYHuang) [reply ci/build.sh:116]: Sorry, CPACK_RPM_PACKAGE_ARCHITECTURE should not be used for DEB.

Comment (TheAssassin) [reply ci/build.sh:116]: I think it's generally easier to read to just `export` those variables. The `DEB` generator won't mind this environment variable.

Comment (MrMYHuang) [reply ci/build.sh]: Done. Please check the latest commit. Thank you.

Comment (TheAssassin) [reply ci/build.sh]: FYI, I receive notifications on every change, no need to comment every time.

**Review by TheAssassin — APPROVED** [review]

**Comment by MrMYHuang** [comment]
One CI job failed with a temporary network problem. Please help me to rerun it. Thanks.

**Comment by TheAssassin** [comment]
We can ignore that. I just want the ARM builds to finish before merging.

File: ci/build.sh (hunk 1 above)
Comment (MrMYHuang) [reply ci/build.sh]: Unfortunately, the generated rpm is arm64, not aarch64:
https://github.com/TheAssassin/AppImageLauncher/suites/6422367770/artifacts/234606851
I am studying why...

**Comment by TheAssassin** [comment]
Still broken:

```
   Date      Time    Attr         Size   Compressed  Name
------------------- ----- ------------ ------------  ------------------------
2022-05-09 00:53:56 .....      3845088      3821973  appimagelauncher-2.2.0-gha97~9165d4f.arm64.rpm
2022-05-09 00:39:04 .....      4023604      4000514  appimagelauncher-2.2.0-gha97~9165d4f.armhf.rpm
2022-05-08 23:35:24 .....      2272428      2249088  appimagelauncher-2.2.0-gha97~9165d4f.i386.rpm
2022-05-08 23:34:32 .....      2321116      2297670  appimagelauncher-2.2.0-gha97~9165d4f.x86_64.rpm
2022-05-09 00:53:56 .....      3826384      3827554  appimagelauncher-9165d4f.source.tar.xz
2022-05-08 23:32:26 .....     51466288     50984929  appimagelauncher-lite-2.2.0-gha97-9165d4f-i386.AppImage
2022-05-08 23:34:32 .....      4012254      4013440  appimagelauncher_2.2.0-gha97~9165d4f+bionic_amd64.deb
2022-05-09 00:53:58 .....      3976796      3977975  appimagelauncher_2.2.0-gha97~9165d4f+bionic_arm64.deb
2022-05-09 00:39:04 .....      4153778      4155014  appimagelauncher_2.2.0-gha97~9165d4f+bionic_armhf.deb
2022-05-08 23:35:24 .....      4077576      4078792  appimagelauncher_2.2.0-gha97~9165d4f+bionic_i386.deb
------------------- ----- ------------ ------------  ------------------------
2022-05-09 00:53:58           83975312     83406949  10 files
```

I suppose these variables you try to set are not read from the environment at all, but I'm not sure about it.

For `x86_64` I've chosen to use the CPack CMake files, which I think is the way to go here as well:
https://github.com/TheAssassin/AppImageLauncher/blob/77fc3e55161c163b4a7bfaf36ebc94c0e64a835d/cmake/cpack_rpm.cmake#L4-L12
https://github.com/TheAssassin/AppImageLauncher/blob/77fc3e55161c163b4a7bfaf36ebc94c0e64a835d/cmake/cpack_deb.cmake#L31-L39

What do you think about removing the change from the shell script again and moving the fix into (both) those files?

**Comment by MrMYHuang** [comment]
So sorry, CI build of the commit 8fa52d1 correctly generates aarch64 rpm (it works), but incorrectly aarch64 deb.

I have no idea why the commit causes deb arch changing from arm64 to aarch64. However, I forcely set CPACK_DEBIAN_PACKAGE_ARCHITECTURE to arm64 as suggested by @TheAssassin in the latest commit:
```
    elseif(CPACK_DEBIAN_PACKAGE_ARCHITECTURE MATCHES "aarch64")
        set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "arm64")
```

**Comment by MrMYHuang** [comment]
I confirm the latest commit works. It builds aarch64 rpm and arm64 deb. They work well on Fedora aarch64 and Ubuntu arm64.

**Comment by MrMYHuang** [comment]
> I confirm the latest commit works. It builds aarch64 rpm and arm64 deb. They work well on Fedora aarch64 and Ubuntu arm64.

Hello @TheAssassin, could merge https://github.com/TheAssassin/AppImageLauncher/pull/510/commits/175dce647d4622332cd14647da8d97127b8ec436 to master branch? Thanks.

**Review by TheAssassin — APPROVED** [review]

**Pull request merged**